rpcs3/rpcs3qt/gui_settings.cpp

```diff
@@ -243,6 +243,15 @@ QStringList gui_settings::GetStylesheetEntries()
 	{
 		res.append(entry.baseName());
 	}
+#ifdef __linux__
+	// Makes stylesheets load if using AppImage or installed to /usr/bin
+	QDir linuxStylesheetDir = QCoreApplication::applicationDirPath() + "/../share/rpcs3/GuiConfigs/";
+	QFileInfoList linux_entries = linuxStylesheetDir.entryInfoList(nameFilter, QDir::Files);
+	for (const QFileInfo &entry : linux_entries)
+	{
+		res.append(entry.baseName());
+	}
```

Comment:
I just noticed these code blocks were already implemented as a reuseable function in `gui::utils::get_dir_entries` in qt_utils.cpp

you might just use: `res.append(gui::utils::get_dir_entries(dir, nameFilter));`

same goes for the guiconfigs part above.

(If it makes sense to do that)

Reply:
Adressed